modules/shared/partials/acid-transactions.adoc

```diff
@@ -301,51 +298,28 @@ This provides these benefits:
 - It takes care of issuing `BEGIN TRANSACTION`, `END TRANSACTION`, `COMMIT` and `ROLLBACK` automatically.  These become an implementation detail and you should not use these statements inside the lambda.
 
 === Supported N1QL
+
 The majority of N1QL DML statements are permitted within a transaction.  Specifically: INSERT, UPSERT, DELETE, UPDATE, MERGE and SELECT are supported.
 
 DDL statements, such as CREATE INDEX, are not.
 // end::query-intro[]
 
 
-// tag::query-options[]
-The supported options are:
-
-* parameters
-* scanConsistency
-* flexIndex
-* serializer
-* clientContextId
-* scanWait
-* scanCap
-* pipelineBatch
-* pipelineCap
-* profile
-* readonly
-* adhoc
-* raw
-// end::query-options[]
-
-
 // tag::query-perf[]
-=== Query Concurrency
-Only one query statement will be performed by the query service at a time.
-The reactive API allows performing multiple concurrent query statements, but this may result internally in some added network traffic due to retries, and is unlikely to provide any increased performance.
```

Comment:
I think we could keep this by referring to concurrent ops (which are possible with any of the implementations) more abstractly.